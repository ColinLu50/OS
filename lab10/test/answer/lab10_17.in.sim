12091
450
278
240
278
240
