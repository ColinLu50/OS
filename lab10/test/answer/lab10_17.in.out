FCFS
36 -> 97 -> 10 -> 139 -> 188 -> 138 -> 147 -> 3 -> 80 -> 228 -> 158 -> 80 -> 220 -> 134 -> 0 -> 190 -> 70 -> 218 -> 26 -> 194 -> 44 -> 35 -> 150 -> 233 -> 148 -> 132 -> 133 -> 98 -> 101 -> 117 -> 86 -> 113 -> 123 -> 213 -> 147 -> 35 -> 38 -> 136 -> 13 -> 57 -> 139 -> 194 -> 25 -> 195 -> 160 -> 68 -> 239 -> 115 -> 50 -> 32 -> 175 -> 9 -> 109 -> 236 -> 151 -> 63 -> 27 -> 173 -> 91 -> 13 -> 90 -> 226 -> 143 -> 5 -> 72 -> 164 -> 174 -> 220 -> 213 -> 207 -> 220 -> 123 -> 13 -> 98 -> 175 -> 202 -> 131 -> 213 -> 52 -> 217 -> 178 -> 84 -> 231 -> 60 -> 151 -> 8 -> 194 -> 175 -> 39 -> 29 -> 26 -> 106 -> 211 -> 112 -> 77 -> 81 -> 141 -> 127 -> 112 -> 205 -> 228 -> 238 -> 188 -> 184 -> 8 -> 235 -> 66 -> 192 -> 95 -> 35 -> 28 -> 179 -> 111 -> 80 -> 232 -> 27 -> 191 -> 119 -> 72 -> 109 -> 49 -> 235 -> 14 -> 132 -> 220 -> 178 -> 1 -> 56 -> 36 -> 168 -> 67 -> 54 -> 208 -> 155 -> 135 -> 242 -> 100 -> 185 -> 129 -> 125 -> 176 -> 183
12091
SSTF
36 -> 36 -> 35 -> 35 -> 35 -> 38 -> 39 -> 44 -> 49 -> 50 -> 52 -> 54 -> 56 -> 57 -> 60 -> 63 -> 66 -> 67 -> 68 -> 70 -> 72 -> 72 -> 77 -> 80 -> 80 -> 80 -> 81 -> 84 -> 86 -> 90 -> 91 -> 95 -> 97 -> 98 -> 98 -> 100 -> 101 -> 106 -> 109 -> 109 -> 111 -> 112 -> 112 -> 113 -> 115 -> 117 -> 119 -> 123 -> 123 -> 125 -> 127 -> 129 -> 131 -> 132 -> 132 -> 133 -> 134 -> 135 -> 136 -> 138 -> 139 -> 139 -> 141 -> 143 -> 147 -> 147 -> 148 -> 150 -> 151 -> 151 -> 155 -> 158 -> 160 -> 164 -> 168 -> 173 -> 174 -> 175 -> 175 -> 175 -> 176 -> 178 -> 178 -> 179 -> 183 -> 184 -> 185 -> 188 -> 188 -> 190 -> 191 -> 192 -> 194 -> 194 -> 194 -> 195 -> 202 -> 205 -> 207 -> 208 -> 211 -> 213 -> 213 -> 213 -> 217 -> 218 -> 220 -> 220 -> 220 -> 220 -> 226 -> 228 -> 228 -> 231 -> 232 -> 233 -> 235 -> 235 -> 236 -> 238 -> 239 -> 242 -> 32 -> 29 -> 28 -> 27 -> 27 -> 26 -> 26 -> 25 -> 14 -> 13 -> 13 -> 13 -> 10 -> 9 -> 8 -> 8 -> 5 -> 3 -> 1 -> 0
450
SCAN
36 -> 35 -> 32 -> 29 -> 28 -> 27 -> 26 -> 25 -> 14 -> 13 -> 10 -> 9 -> 8 -> 5 -> 3 -> 1 -> 0 -> 0 -> 38 -> 39 -> 44 -> 49 -> 50 -> 52 -> 54 -> 56 -> 57 -> 60 -> 63 -> 66 -> 67 -> 68 -> 70 -> 72 -> 77 -> 80 -> 81 -> 84 -> 86 -> 90 -> 91 -> 95 -> 97 -> 98 -> 100 -> 101 -> 106 -> 109 -> 111 -> 112 -> 113 -> 115 -> 117 -> 119 -> 123 -> 125 -> 127 -> 129 -> 131 -> 132 -> 133 -> 134 -> 135 -> 136 -> 138 -> 139 -> 141 -> 143 -> 147 -> 148 -> 150 -> 151 -> 155 -> 158 -> 160 -> 164 -> 168 -> 173 -> 174 -> 175 -> 176 -> 178 -> 179 -> 183 -> 184 -> 185 -> 188 -> 190 -> 191 -> 192 -> 194 -> 195 -> 202 -> 205 -> 207 -> 208 -> 211 -> 213 -> 217 -> 218 -> 220 -> 226 -> 228 -> 231 -> 232 -> 233 -> 235 -> 236 -> 238 -> 239 -> 242
278
C-SCAN
36 -> 35 -> 32 -> 29 -> 28 -> 27 -> 26 -> 25 -> 14 -> 13 -> 10 -> 9 -> 8 -> 5 -> 3 -> 1 -> 0 -> 0 -> 242 -> 242 -> 239 -> 238 -> 236 -> 235 -> 233 -> 232 -> 231 -> 228 -> 226 -> 220 -> 218 -> 217 -> 213 -> 211 -> 208 -> 207 -> 205 -> 202 -> 195 -> 194 -> 192 -> 191 -> 190 -> 188 -> 185 -> 184 -> 183 -> 179 -> 178 -> 176 -> 175 -> 174 -> 173 -> 168 -> 164 -> 160 -> 158 -> 155 -> 151 -> 150 -> 148 -> 147 -> 143 -> 141 -> 139 -> 138 -> 136 -> 135 -> 134 -> 133 -> 132 -> 131 -> 129 -> 127 -> 125 -> 123 -> 119 -> 117 -> 115 -> 113 -> 112 -> 111 -> 109 -> 106 -> 101 -> 100 -> 98 -> 97 -> 95 -> 91 -> 90 -> 86 -> 84 -> 81 -> 80 -> 77 -> 72 -> 70 -> 68 -> 67 -> 66 -> 63 -> 60 -> 57 -> 56 -> 54 -> 52 -> 50 -> 49 -> 44 -> 39 -> 38
240
LOOK
36 -> 35 -> 32 -> 29 -> 28 -> 27 -> 26 -> 25 -> 14 -> 13 -> 10 -> 9 -> 8 -> 5 -> 3 -> 1 -> 0 -> 38 -> 39 -> 44 -> 49 -> 50 -> 52 -> 54 -> 56 -> 57 -> 60 -> 63 -> 66 -> 67 -> 68 -> 70 -> 72 -> 77 -> 80 -> 81 -> 84 -> 86 -> 90 -> 91 -> 95 -> 97 -> 98 -> 100 -> 101 -> 106 -> 109 -> 111 -> 112 -> 113 -> 115 -> 117 -> 119 -> 123 -> 125 -> 127 -> 129 -> 131 -> 132 -> 133 -> 134 -> 135 -> 136 -> 138 -> 139 -> 141 -> 143 -> 147 -> 148 -> 150 -> 151 -> 155 -> 158 -> 160 -> 164 -> 168 -> 173 -> 174 -> 175 -> 176 -> 178 -> 179 -> 183 -> 184 -> 185 -> 188 -> 190 -> 191 -> 192 -> 194 -> 195 -> 202 -> 205 -> 207 -> 208 -> 211 -> 213 -> 217 -> 218 -> 220 -> 226 -> 228 -> 231 -> 232 -> 233 -> 235 -> 236 -> 238 -> 239 -> 242
278
C-LOOK
36 -> 35 -> 32 -> 29 -> 28 -> 27 -> 26 -> 25 -> 14 -> 13 -> 10 -> 9 -> 8 -> 5 -> 3 -> 1 -> 0 -> 242 -> 239 -> 238 -> 236 -> 235 -> 233 -> 232 -> 231 -> 228 -> 226 -> 220 -> 218 -> 217 -> 213 -> 211 -> 208 -> 207 -> 205 -> 202 -> 195 -> 194 -> 192 -> 191 -> 190 -> 188 -> 185 -> 184 -> 183 -> 179 -> 178 -> 176 -> 175 -> 174 -> 173 -> 168 -> 164 -> 160 -> 158 -> 155 -> 151 -> 150 -> 148 -> 147 -> 143 -> 141 -> 139 -> 138 -> 136 -> 135 -> 134 -> 133 -> 132 -> 131 -> 129 -> 127 -> 125 -> 123 -> 119 -> 117 -> 115 -> 113 -> 112 -> 111 -> 109 -> 106 -> 101 -> 100 -> 98 -> 97 -> 95 -> 91 -> 90 -> 86 -> 84 -> 81 -> 80 -> 77 -> 72 -> 70 -> 68 -> 67 -> 66 -> 63 -> 60 -> 57 -> 56 -> 54 -> 52 -> 50 -> 49 -> 44 -> 39 -> 38
240
